src/main/resources/swagger.api/mosaic.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.0
+info:
+  title: Mosaic integration API
+  description: "Mosaic integration API"
+  version: 0.0.1
+servers:
+  - url: /mosaic
+paths:
+  /orders:
+    description: Orders API
+    post:
+      summary: Create a new order in FOLIO
+      operationId: createMosaicOrder
+      requestBody:
+        description: Mosaic Order object
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/MosaicOrder"
+      responses:
+        '201':
+          description: Order successfully created, returns the POL number
+          content:
+            text/plain:
+              schema:
+                type: string
+              example: "10001-1"
+        '400':
+          description:
+            Bad request, e.g. malformed request body or query parameter.
+            Details of the error (e.g. name of the parameter or line/character number
+            with malformed data) provided in the response.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/errors"
+        '500':
+          description: Internal server error, e.g. due to misconfiguration
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/errors"
```

Comment:
small suggestion to make them reusable
here is example
https://github.com/folio-org/mod-consortia/blob/d101eefdcf2d1f560f21c12a995c4170b682e0f0/src/main/resources/swagger.api/user_tenants.yaml#L111

```
      responses:
        "200":
          $ref: "#/components/responses/UserTenantCollection"
        "400":
          $ref: "#/components/responses/BadRequest"
        "404":
          $ref: "#/components/responses/NotFound"
        "500":
          $ref: "#/components/responses/InternalServerError"
```

Reply:
will be addressed in subsequent pr, because we were needed to merge this PR urgently to fix build issue